analyzers/src/SonarAnalyzer.Common/Json/Parsing/Enums.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.Json.Parsing
+{
+    public enum Kind
+    {
+        Object,
+        List,
+        Value
+    }
+
+    internal enum Symbol
+    {
+        EOI,
+        // Special
+        OpenCurlyBracket,
+        CloseCurlyBracket,
+        OpenSquareBracket,
+        CloseSquareBracket,
+        Comma,
+        Colon,
+        // Terms
+        Value   // String, Number (Integer, Decimal, Double), Null, True, False
```

Comment:
Is this intended as documentation? (If so, why not use `/// <summary>` or as TODO? (in that case, mark it as such).

Reply:
This is just an explanatory comment for me